### PR TITLE
Formatters options

### PR DESCRIPTION
**Pull request by Oreganon**
Requested by some grumpy boomer.

One bug/feature: without emote formatter comboed emotes are not displayed.

### CODE REVIEW

File: assets/chat/js/messages.js [hunk 1]
```diff
@@ -17,13 +17,22 @@ const MessageTypes = {
...
-const formatters = new Map()
+var formatters = new Map()
 formatters.set('html', new HtmlTextFormatter())
 formatters.set('url', new UrlFormatter())
 formatters.set('emote', new EmoteFormatter())
 formatters.set('mentioned', new MentionedUserFormatter())
 formatters.set('green', new GreenTextFormatter())
 
+exports.setFormattersFromSettings = function(settings) {
+    formatters = new Map();
```
Comment (KenanY) [inline assets/chat/js/messages.js]: Please use [`Map#clear()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/clear) instead.

File: assets/chat/js/chat.js [hunk 2]
```diff
@@ -509,6 +514,11 @@ class Chat {
 
         // Update maxlines
         [...this.windows].forEach(w => w.maxlines = this.settings.get('maxlines'));
+
+        // Formatter enable/disable
+        var messages = require('./messages.js');
```
Comment (KenanY) [inline assets/chat/js/chat.js]: `const`?

File: assets/chat/js/messages.js [hunk 3]
```diff
@@ -17,13 +17,22 @@ const MessageTypes = {
     USER      : 'USER',
     EMOTE     : 'EMOTE'
 }
-const formatters = new Map()
+var formatters = new Map()
```
Comment (KenanY) [inline assets/chat/js/messages.js]: Can be `const` again.

**Comment by xDashh** [comment]
merged in https://github.com/MemeLabs/chat-gui/commit/15f05feca609dd40b8cf44aeb6e31ef2a6b51ae1 thanks!

**Pull request closed without merging**